docs/use/grammar.rst

```diff
@@ -331,7 +347,7 @@ Keywords
 * ``gate``: Gate declaration or definition.
```

Comment:
Do we need one here for constants?

Reply:
`* ``end``: End of a definition or options section.`
This now included constants. Do we have a general term for these?

Reply:
Good catch. I'll add this! 👍